opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -380,7 +669,7 @@ namespace Opm {
             {
                 // assembles the well equations and applies the wells to
                 // the reservoir equations as a source term.
-                wellModel().assemble(iterationIdx, dt);
+                wellModel().assemble(iterationIdx, false, dt);
```

Comment:
the additional parameter is to avoid well solves which change the state. This is similar to the problems with deserializing, serializing and restart in ewoms. (The code was written for foward mode only, and so is the consepts.)